dashboard/dashboard.js

```diff
@@ -109,6 +109,10 @@ module.exports = (client) => {
   // Initializes passport and session.
   app.use(passport.initialize());
   app.use(passport.session());
+  
+  // Content Security Policy (CSP) is an added layer of security that helps to detect and mitigate certain types of attacks
```

Comment:
Really not sure why this chunk of text was added...

Reply:
reduced the text in next commit, this here because, in some browsers 3rd party usages and links are blocked because of helmet()